developing_SoCCo.py

```diff
@@ -0,0 +1,471 @@
+
```

Comment:
@jonathanrocher: This is my code that I use to for development and testing of changes.  Not intended for public viewing.  Could move it out of this folder....